site2/docs/developing-binary-protocol.md

```diff
@@ -406,12 +406,19 @@ message CommandAck {
 ```
 
 Parameters:
- * `consumer_id` → Id of an already established consumer
+ * `consumer_id` → ID of an already established consumer
  * `ack_type` → Type of acknowledgment: `Individual` or `Cumulative`
- * `message_id` → Id of the message to acknowledge
+ * `message_id` → ID of the message to acknowledge
  * `validation_error` → *(optional)* Indicates that the consumer has discarded
    the messages due to: `UncompressedSizeCorruption`,
    `DecompressionError`, `ChecksumMismatch`, `BatchDeSerializeError`
+ * `properties` -> *(optional)* Reserved configuration items
+ * `txnid_most_bits` -> *(optional)* Same as TC ID, `txnid_most_bits` and `txnid_least_bits`
+   uniquely identify a transaction.
+ * `txnid_least_bits` -> *(optional)* The id of the transaction opened in a TC,
```

Comment:
you miss this occurrence

Reply:
This is the id, not the Id, also need to be changed?